aten/src/ATen/core/DeprecatedTypeProperties.h

```diff
@@ -19,8 +19,8 @@ class Tensor;
 // dtype-specific.
 class CAFFE2_API DeprecatedTypeProperties {
  public:
-  DeprecatedTypeProperties(Backend backend, ScalarType scalar_type, bool is_variable)
-    : backend_(backend), scalar_type_(scalar_type), is_variable_(is_variable) {}
+  DeprecatedTypeProperties(Backend backend, ScalarType scalar_type)
```

Comment:
wait, is it true that we haven't actually deprecated "DeprecatedTypeProperties" (i.e. via the public accessor, `tensor.type()`?

Reply:
I think we forgot to put the deprecated marker on it. See also https://github.com/pytorch/pytorch/pull/25306

Reply:
Thanks!  I filed https://github.com/pytorch/pytorch/issues/29161.